apps/src/sites/studio/pages/projects/public.js

```diff
@@ -25,6 +28,11 @@ $(document).ready(() => {
           subHeadingText={i18n.projectsSubHeading()}
           short={true}
         />
+        <StartNewProject
+          isRtl={isRtl}
+          canViewFullList={true}
+          canViewAdvancedTools={true}
```

Comment:
Nit: Prefer providing boolean literal props as flags
```jsx
<StartNewProject
  isRtl={isRtl}
  canViewFullList
  canViewAdvancedTools
/>
```
(according to AirBnB's React/JSX styleguide, [second rule in the Props section](https://github.com/airbnb/javascript/tree/master/react#props))

Reply:
TIL, thanks Brad!